pkg/addons/data/CronJob.batch/openshift-etcd/etcd-backup.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: etcd-backup
+  namespace: openshift-etcd
+spec:
+  # todo: move to 1 hour after testing
+  schedule: "*/10 * * * *"
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          serviceAccountName: etcd-operator
```

Comment:
The etcd-operator has elevated access where this cronjob only needs to create an etcdbackup. Needs its own service account, role, and rolebinding.